package.json

```diff
@@ -94,13 +94,14 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "create-react-context": "^0.3.0",
+    "date-fns": "^2.30.0",
     "element-resize-detector": "^1.1.12",
     "lodash.isequal": "^4.5.0",
     "memoize-one": "^5.1.1"
   },
   "peerDependencies": {
     "interactjs": "^1.3.4",
-    "moment": "*",
+    "date-fns": "*",
```

Comment:
why "*" and not "^2.30.0"?

Reply:
[fix date fns dependency version](https://github.com/Optiways/react-calendar-timeline/pull/1/commits/99589d5c2f8b816107d72cfd6cfcb3049010b803)